src/Faker/Provider/hr_HR/Person.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Faker\Provider\hr_HR;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @link http://www.behindthename.com/top/lists/croatia/2009
+     */
+    protected static $firstNameMale = array(
+        'Luka', 'Ivan', 'Marko', 'David', 'Filip', 'Josip', 'Petar', 'Karlo', 'Matej', 'Ivano', 'Roko', 'Leon',
```

Comment:
Please sort the names by alphabetical order. Easier to maintain. Same for other name lists.